src/app/containers/Ad/Amp/__snapshots__/index.test.jsx.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`AMP Ads Snapshots should correctly render an AMP leaderboard ad 1`] = `
 .c0 {
-  background-color: #F8F8F8;
+  background-color: #F6F6F6;
```

Comment:
Weird... not sure why these snapshots have been updated - we might be a wee bit out of date